lib/python/pcm_placeholder.py.in

```diff
@@ -15,4 +15,4 @@ that has to be configured by CMake
 
 """
 
-PCMSOLVER_PARSE_DIR='@PCMSOLVER_PARSE_DIR@'
+PCMSolver_PARSE_DIR='@PCMSolver_PARSE_DIR@'
```

Comment:
@loriab yes, this glitch slipped through.
`PCMSolver_PARSE_DIR` is defined as `${INSTALL_DIR}/bin` aka `${PROJECT_BINARY_DIR}/interfaces/pcmsolver/install/bin` if the code gets compiled on the spot. If the library was compiled somewhere else, the value `${PCMSOLVER_ROOT}/bin` is used. In the former case, we should install to the global install prefix. Problem is: the install step always happens for CMake subprojects and the default install prefix requires root privileges...